cmd/ig/main.go

```diff
@@ -106,6 +110,15 @@ func main() {
 	rootCmd.AddCommand(common.NewRunCommand(rootCmd, runtime, hiddenColumnTags, common.CommandModeRun))
 	rootCmd.AddCommand(common.NewConfigCmd(runtime, rootFlags))
 
+	PprofAddr, _ := rootCmd.PersistentFlags().GetString("pprof-addr")
```

Comment:
```suggestion
	pprofAddr, _ := rootCmd.PersistentFlags().GetString("pprof-addr")
```

(lower case, to not confuse with exported variable)